### DISCUSSION TITLE
BED_MESH_PROFILE LOAD=default

### DISCUSSION THREAD

**Discussion by dev-zetta** [Guilouz/Klipper-Flsun-Speeder-Pad#112] [Q&A]
Hello Cyril,

just a dumb question, but I don't see in START_PRINT macros a command to load the bed mesh profile:

BED_MESH_PROFILE LOAD=default

Is it loaded automatically?

**Comment by Guilouz** [comment]
Yes it's loaded by a delayed macro at Klipper startup:

[delayed_gcode bed_mesh_init]
initial_duration: .01
gcode:
  BED_MESH_PROFILE LOAD=default
